homeassistant/components/sensor/rfxtrx.py

```diff
@@ -71,14 +71,16 @@ def sensor_update(event):
 
         if device_id in rfxtrx.RFX_DEVICES:
             sensors = rfxtrx.RFX_DEVICES[device_id]
-            for key in sensors:
-                sensor = sensors[key]
+            for data_type in sensors:
+                if data_type not in event.values:
```

Comment:
Could you add a comment to describe this?

Reply:
OK, done